script/openqa-bootstrap

```diff
@@ -39,6 +39,7 @@ fi
 
 
 # setup database
+chown -R postgres: /var/lib/pgsql/ # fix broken postgres working dir permissions in the nspawn container
```

Comment:
and does this problem not appear on non-nspawn setups? Is this a problem with the postgresql package then?